install/roles/extra-certs/tasks/main.yml

```diff
@@ -6,6 +6,6 @@
     apply:
       tags: certificates
   with_items:
-    - '{{ extra_certs }}'
+    - '{{ extra_certs | default([]) }}'
```

Comment:
Maybe keep this change for another commit.

Reply:
idem

Reply:
I agree with the previous one, but this one seems needed to pass the build, or the load-defaults role does not work.